docs/usage/opentelemetry.md

```diff
@@ -0,0 +1,27 @@
+---
+title: OpenTelemetry
+description: How to use OpenTelemetry with Renovate
+---
+
+# OpenTelemetry and Renovate
+
+Renovate supports OpenTelemetry which is an emerging monitoring standard.
+
+OpenTelemetry supports three types of observability data:
+
+- traces
+- metrics
+- logs
+
+Renovate uses only traces which are exported using OpenTelemetryProtocol ( OTLP ), other export methods are not supported.
```

Comment:
What does that mean in practice? e.g. as someone unfamiliar with OT, I think "export by who?"

Reply:
Is the (applied) suggestion of @Shegox under stable well enough for Non-OT users?